oar/cli/cmd_create_test_report.py

```diff
@@ -0,0 +1,28 @@
+import click
+import logging
+import oar.core.util as util
+from oar.core.worksheet_mgr import WorksheetManager, WorksheetException
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.pass_context
+def create_test_report(ctx):
+    """
+    Create test report for z-stream release
+    """
+    # get config store from context
+    cs = ctx.obj["cs"]
+    # init worksheet manager
+    try:
+        wm = WorksheetManager(cs)
+        report = wm.create_test_report()
+    except WorksheetException as we:
```

Comment:
@rioliu-rh how about other exception?

Reply:
catch all the exception in `create_test_report` raise as `WorksheetException`